src/AutoRest.CSharp.V3/Output/Builders/ClientBuilder.cs

```diff
@@ -5,68 +5,86 @@
 using System.Collections.Generic;
 using System.Diagnostics;
 using System.Linq;
-using AutoRest.CSharp.V3.ClientModels.Serialization;
-using AutoRest.CSharp.V3.Pipeline.Generated;
-using AutoRest.CSharp.V3.Plugins;
+using AutoRest.CSharp.V3.Input;
+using AutoRest.CSharp.V3.Input.Generated;
+using AutoRest.CSharp.V3.Output.Models;
+using AutoRest.CSharp.V3.Output.Models.Requests;
+using AutoRest.CSharp.V3.Output.Models.Responses;
+using AutoRest.CSharp.V3.Output.Models.Serialization;
+using AutoRest.CSharp.V3.Output.Models.Shared;
+using AutoRest.CSharp.V3.Output.Models.TypeReferences;
 using AutoRest.CSharp.V3.Utilities;
 using Azure.Core;
-using HttpHeader = AutoRest.CSharp.V3.Pipeline.Generated.HttpHeader;
+using HttpHeader = AutoRest.CSharp.V3.Input.Generated.HttpHeader;
+using Parameter = AutoRest.CSharp.V3.Output.Models.Shared.Parameter;
+using Request = AutoRest.CSharp.V3.Output.Models.Requests.Request;
+using Response = AutoRest.CSharp.V3.Output.Models.Responses.Response;
+using SerializationFormat = AutoRest.CSharp.V3.Output.Models.Serialization.SerializationFormat;
 
-namespace AutoRest.CSharp.V3.ClientModels
+namespace AutoRest.CSharp.V3.Output.Builders
 {
     internal class ClientBuilder
     {
-        public static ServiceClient BuildClient(OperationGroup arg)
+        public static Client BuildClient(OperationGroup operationGroup)
         {
-            List<ClientMethod> methods = new List<ClientMethod>();
-            Dictionary<string, ServiceClientParameter> clientParameters = new Dictionary<string, ServiceClientParameter>();
-
-            var allClientParameters = arg.Operations
+            var allClientParameters = operationGroup.Operations
                 .SelectMany(op => op.Request.Parameters)
                 .Where(p => p.Implementation == ImplementationLocation.Client)
                 .Distinct();
-
+            Dictionary<string, Parameter> clientParameters = new Dictionary<string, Parameter>();
             // Deduplication required because of https://github.com/Azure/autorest.modelerfour/issues/100
-            foreach (Parameter clientParameter in allClientParameters)
+            foreach (Input.Generated.Parameter clientParameter in allClientParameters)
```

Comment:
No full qualification should be required anywhere.

Reply:
![image](https://user-images.githubusercontent.com/17788297/72466275-6dc89600-378d-11ea-9870-1d6ddc76df5f.png)

Reply:
Right, let's not have multiple types with a same name in solution.

Reply:
Done